README.md

```diff
@@ -25,27 +24,13 @@ To build the project execute the following command:
 
 ### Running the application
 
-Create the image of the application by executing the following command:
+The easiest way to run the application locally is to use the `bootWithCCD` Gradle task.
 
```

Comment:
Might be good to add the az logins here, as they are needed to pull the images for the first time:
```
az acr login --name hmctsprivate --subscription DCD-CNP-PROD
az acr login --name hmctspublic --subscription DCD-CNP-PROD
```

Reply:
updated